### PR TITLE
Add ESLint and Prettier

### PR DESCRIPTION
**Pull request by QuantumManiac**
Added ESLint and Prettier. Also formatted all the code and fixed a few errors that it picked up.

- - -
This change is [Reviewable](https://reviewable.io/reviews/waterloo-rocketry/minerva/77)

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+    "editor.rulers": [
+        120
```
Comment (QuantumManiac) [inline .vscode/settings.json:3]: Chose 120 lines. Thoughts on 80 instead?

**Review by KavinSatheeskumar — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/waterloo-rocketry/minerva/77)* status: 0 of 21 files reviewed, 3 unresolved discussions (waiting on @BluCodeGH, @chrissank, and @QuantumManiac)

___
*[`.vscode/settings.json` line 3 at r1](https://reviewable.io/reviews/waterloo-rocketry/minerva/77#gh-1159093006:-NSSItNiBDpakWNcFs2J:b4wob1k) ([raw file](https://github.com/waterloo-rocketry/minerva/blob/c5e90d5ee6178deef7d9bf4d4a2a41ae9cf34bc1/.vscode/settings.json#L3)):*
<details><summary><i>Previously, QuantumManiac (Chamath Wijesekera) wrote…</i></summary><blockquote>

Chose 120 lines. Thoughts on 80 instead?
</blockquote></details>

I am a long line enjoyer, so 120 is fine
___
*[`infra/minerva.js` line 3 at r1](https://reviewable.io/reviews/waterloo-rocketry/minerva/77#-NSSJILg9sZDoRpJyWyh:-NSSJILg9sZDoRpJyWyi:b-20mx4r) ([raw file](https://github.com/waterloo-rocketry/minerva/blob/c5e90d5ee6178deef7d9bf4d4a2a41ae9cf34bc1/infra/minerva.js#L3)):*
> ```JavaScript
> #!/usr/bin/env node
> 
> const cdk = require("aws-cdk-lib");
> ```

I prefer single quotes for js
___
*[`src/index.js` line 74 at r1](https://reviewable.io/reviews/waterloo-rocketry/minerva/77#-NSSJ3vT6P0QcEDApj9g:-NSSJ3vT6P0QcEDApj9h:bikqece) ([raw file](https://github.com/waterloo-rocketry/minerva/blob/c5e90d5ee6178deef7d9bf4d4a2a41ae9cf34bc1/src/index.js#L74)):*
> ```JavaScript
> 
>             await slack.postEphemeralMessage(
>                 "Command failed: " +
> ```

can we use javascript formatted strings?

**Review by QuantumManiac — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/waterloo-rocketry/minerva/77)* status: 0 of 22 files reviewed, 2 unresolved discussions (waiting on @BluCodeGH, @chrissank, and @KavinSatheeskumar)

___
*[`infra/minerva.js` line 3 at r1](https://reviewable.io/reviews/waterloo-rocketry/minerva/77#-NSSJILg9sZDoRpJyWyh:-NSWIUdZ5dM8hhzuefMr:b-4hnumm) ([raw file](https://github.com/waterloo-rocketry/minerva/blob/c5e90d5ee6178deef7d9bf4d4a2a41ae9cf34bc1/infra/minerva.js#L3)):*
<details><summary><i>Previously, KavinSatheeskumar (Kavin Satheeskumar) wrote…</i></summary><blockquote>

I prefer single quotes for js
</blockquote></details>

I would prefer double-quotes due to the frequent use of apostrophes in text, e.g. `"I'm a cool person"` vs `'I\'m a cool person'`. Doesn't matter too much because we have templated strings (\`I'm a cool person\`) but might as well.
___
*[`src/index.js` line 74 at r1](https://reviewable.io/reviews/waterloo-rocketry/minerva/77#-NSSJ3vT6P0QcEDApj9g:-NSWIuNPB2bgkLyuMJiP:bvy2j24) ([raw file](https://github.com/waterloo-rocketry/minerva/blob/c5e90d5ee6178deef7d9bf4d4a2a41ae9cf34bc1/src/index.js#L74)):*
<details><summary><i>Previously, KavinSatheeskumar (Kavin Satheeskumar) wrote…</i></summary><blockquote>

can we use javascript formatted strings?
</blockquote></details>

Yeah sure, I can make that change real quick

**Review by yxyyeah — APPROVED** [review]
> Reviewed 15 of 21 files at r1, 7 of 7 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/waterloo-rocketry/minerva/77)* status: all files reviewed, 2 unresolved discussions (waiting on @chrissank, @KavinSatheeskumar, and @kinaz-xxr)

**Review by QuantumManiac — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/waterloo-rocketry/minerva/77)* status: 14 of 23 files reviewed, all discussions resolved (waiting on @chrissank, @KavinSatheeskumar, @kinaz-xxr, and @yxyyeah)

___
*[`src/index.js` line 74 at r1](https://reviewable.io/reviews/waterloo-rocketry/minerva/77#-NSSJ3vT6P0QcEDApj9g:-NZmq7JdABLwcmmkqdFk:bb6un1j) ([raw file](https://github.com/waterloo-rocketry/minerva/blob/c5e90d5ee6178deef7d9bf4d4a2a41ae9cf34bc1/src/index.js#L74)):*
<details><summary><i>Previously, QuantumManiac (Chamath Wijesekera) wrote…</i></summary><blockquote>

Yeah sure, I can make that change real quick
</blockquote></details>

Done

**Review by slightlyskepticalpotat — APPROVED** [review]
> Reviewed 9 of 21 files at r1, 3 of 7 files at r2, 9 of 10 files at r3, 2 of 2 files at r4, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/waterloo-rocketry/minerva/77)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @chrissank, @KavinSatheeskumar, and @kinaz-xxr)

**Review by Darth-Raazi — APPROVED** [review]
Nitpick formatting changes, non-blocking

> Reviewed all commit messages.
> *[Reviewable](https://reviewable.io/reviews/waterloo-rocketry/minerva/77#-:-NfJD5IJCdiz6ch2V--Q:bl5dqbg)* status: all files reviewed, 3 unresolved discussions (waiting on @chrissank, @KavinSatheeskumar, @kinaz-xxr, and @QuantumManiac)

___
*[`.eslintignore` line 4 at r4](https://reviewable.io/reviews/waterloo-rocketry/minerva/77#-NfJCQFq6xMKEDVsyimc:-NfJCQFq6xMKEDVsyimd:b-k6ip4e) ([raw file](https://github.com/waterloo-rocketry/minerva/blob/eb4b340f22922322d6d526d5e402f7724aa8a992/.eslintignore#L4)):*
> ```ignore
> node_modules
> # do not lint build/ or dist/ folder
> cdk.out
> ```

End files with new lines please
___
*[`.vscode/extensions.json` line 14 at r4](https://reviewable.io/reviews/waterloo-rocketry/minerva/77#-NfJCfHF2NE9IRjfOMTQ:-NfJCfHF2NE9IRjfOMTR:b-ldg2nt) ([raw file](https://github.com/waterloo-rocketry/minerva/blob/eb4b340f22922322d6d526d5e402f7724aa8a992/.vscode/extensions.json#L14)):*
> ```JSON
> 		
> 	]
> }
> ```

See above. If this is an auto-generated file then ignore
___
*[`.vscode/settings.json` line 5 at r4](https://reviewable.io/reviews/waterloo-rocketry/minerva/77#-NfJCp_d9ahPmRkX_WvN:-NfJCp_d9ahPmRkX_WvO:b-34vmyd) ([raw file](https://github.com/waterloo-rocketry/minerva/blob/eb4b340f22922322d6d526d5e402f7724aa8a992/.vscode/settings.json#L5)):*
> ```JSON
>         120
>     ],
> }
> ```

See above

**Review by slightlyskepticalpotat — APPROVED** [review]
> *[Reviewable](https://reviewable.io/reviews/waterloo-rocketry/minerva/77)* status: all files reviewed, 3 unresolved discussions (waiting on @chrissank, @Darth-Raazi, @KavinSatheeskumar, @kinaz-xxr, and @QuantumManiac)

___
*[`.vscode/extensions.json` line 14 at r4](https://reviewable.io/reviews/waterloo-rocketry/minerva/77#-NfJCfHF2NE9IRjfOMTQ:-NfJE0F17Y239KOUBSCu:b-86dvba) ([raw file](https://github.com/waterloo-rocketry/minerva/blob/eb4b340f22922322d6d526d5e402f7724aa8a992/.vscode/extensions.json#L14)):*
<details><summary><i>Previously, Darth-Raazi (Ozayr Raazi) wrote…</i></summary><blockquote>

See above. If this is an auto-generated file then ignore
</blockquote></details>

It is

**Review by slightlyskepticalpotat — APPROVED** [review]
> *[Reviewable](https://reviewable.io/reviews/waterloo-rocketry/minerva/77)* status: all files reviewed, 3 unresolved discussions (waiting on @chrissank, @Darth-Raazi, @KavinSatheeskumar, @kinaz-xxr, and @QuantumManiac)

**Comment by QuantumManiac** [comment]
Closing as development on this project has ceased. Merging a PR that has a massive amount of changes due to linting and formatting also looks to be a headache given that some code has been added between this PR opening and now, and therefore the merging is a bit confused.

**Pull request closed without merging**